wdqs-frontend/latest/entrypoint.sh

```diff
@@ -1,6 +1,17 @@
 #!/bin/sh
 # This file is provided by the wikibase/wdqs-frontend docker image.
 
+# Test if required environment variables have been set
+if [ -z "$WIKIBASE_HOST" ]; then
+echo "WIKIBASE_HOST is required but isn't set. You can pass it using either plain docker or docker-compose";
```

Comment:
I don't see why we really need to list ways to pass it in here.
Docker is what needs the env var, the orchestration probably shouldn't be mentioned here, otherwise we should be listing them all!

Reply:
Sure, I'll remove the docker-compose bit